linode/manager
# fix: [M3-7369] - Safari Launch Darkly MSW Errors

## Description 📝
- Makes Safari useable for development
  - Updates the MSW to fix the bug where api calls would fail if "Disable caches" was on
  - Updates the MSW to fix the bug where Launch Darkly would spam the console with errors
- Performs other clean up and simplification
- Removes broken mock data tool
  - No harm here because it is broken
  - I have plans to reimplement something similar with React Query


## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-11-01 at 12 09 18 PM](https://github.com/linode/manager/assets/115251059/0346c03b-aff0-4100-bdda-447868a0fe01) | ![Screenshot 2023-11-01 at 12 51 06 PM](https://github.com/linode/manager/assets/115251059/c7e98712-1fc8-4deb-b1a8-15806d32aea5) |

## How to test 🧪

### Prerequisite steps 
- Run `yarn` in the repo's root to make sure dependencies are up to date

### Verification steps 
- Open http://localhost:3000 in Safari and verify the console is not flooded with errors
- Verify the MSW still works as expected
- Verify all CI and e2es pass

## As an Author I have considered 🤔
